packages/volto/theme/themes/pastanaga/extras/grid.less

```diff
@@ -222,7 +222,7 @@ body.has-toolbar.has-sidebar .drag.gridBlock.handle.wrapper {
       bottom: 0;
       left: 0;
       width: 100%;
-      height: 100%;
```

Comment:
@sneridagh this probably needs to be added back as well

Reply:
I added it back, @dobri1408 please confirm that this was wrong too, or justify why you remove it. thanks!